roles/ssh/tasks/main.yml

```diff
@@ -221,7 +221,7 @@
   ansible.builtin.lineinfile:
     <<: *sshd_config_args
     regexp: "^#?PubkeyAcceptedKeyTypes"
-    line: "PubkeyAcceptedKeyTypes {{ ssh_preset.allowed_pubkey_algos | intersect(queried_pubkey_algos.stdout_lines) | join(',') }}"
+    line: "PubkeyAcceptedKeyTypes {{ ssh_preset.allowed_pubkey_algos | intersect(queried_pubkey_algos.stdout_lines) | sort | join(',') }}"
```

Comment:
*Ideally*, we would keep the order as they are in the preset. Right now, this prefers sha256 over sha512 due to alphabetic sorting.

Reply:
Added `reverse=true` to the sort until this is fixed, as the stronger algorithms are *generally* later in the alphabet. If an algorithm is deemed not secure enough, it should be removed from the list.